src/hydra-evaluator/hydra-evaluator.cc

```diff
@@ -366,6 +366,9 @@ struct Evaluator
                     printInfo("received jobset event");
                 }
 
+            } catch (pqxx::broken_connection & e) {
+                printError("Database connection broken: %s", e.what());
+                std::_Exit(1);
```

Comment:
Isn't there `std::exit` or just `exit`?

Reply:
Those calls exist, yes. I was following other bits in the code where exceptions happen and it is called this way. I don't have a good reason to deviate from that, but I'm happy to change if there is.

Reply:
I don't know, the behavior is different apparently, so stick to the general pattern I'd say.